Rakefile

```diff
@@ -9,7 +9,7 @@ namespace :etcd do
   task :load_admin do
     p 'Generating default Tendrl admin'
     etcd_config = Tendrl.etcd_config(ENV['RACK_ENV'])
-    password = SecureRandom.hex(4)
+    password = SecureRandom.base64(8)
```

Comment:
I'd prefer to simply set it to admin and force a password change on login to the UI